docs/4.x/database-schema.md

```diff
@@ -129,6 +129,7 @@ Visit actions contain the following information:
 - `idaction_name`: the ID of the page title action type for this action
 - `idaction_name_ref`: the ID of the page title action type for the previous action in the visit
 - `time_spent_ref_action`: the amount of time spent doing the previous action in seconds (see below for details)
+- `time_spent`: the amount of time spent on this action (requires the [CustomDimensions plugin](https://plugins.matomo.org/CustomDimensions))
```

Comment:
@mattab maybe don't link to the plugin as it will eventually disappear (since it will be in core)

Reply:
removed the link :+1: